src/components/GradesView/GradebookTable/LabelReplacements.test.jsx

```diff
@@ -16,6 +17,32 @@ jest.mock('@edx/paragon', () => ({
   Tooltip: () => 'Tooltip',
 }));
 
+configureI18n({
```

Comment:
I would suggest updating `setupTest.js`
```javascript
jest.mock('@edx/frontend-platform/i18n', () => {
  const i18n = jest.requireActual('@edx/frontend-platform/i18n');
  const PropTypes = jest.requireActual('prop-types');
  return {
    ...i18n,
    intlShape: PropTypes.shape({
      formatMessage: jest.fn(msg => msg.defaultMessage),
    }),
    defineMessages: m => m,
    FormattedMessage: () => 'FormattedMessage',
    getLocale: jest.fn(),
  };
});
```

Then you can make more verbose snapshot for rtl and ltr overlay.
```javascript
describe('snapshot', () => {
  test('right to left overlay placement', () => {
    getLocale.mockImplementation(() => 'en');
    el = shallow(<TotalGradeLabelReplacement />);
    expect(el).toMatchSnapshot();
  });
  test('left to right overlay placement', () => {
    getLocale.mockImplementation(() => 'ar');
    el = shallow(<TotalGradeLabelReplacement />);

    expect(el).toMatchSnapshot();
  })
});
```

Reply:
And you won't need to define this configureI18n, which I consider inconsistent for everyone's use case.

Reply:
@leangseu-edx Thanks a lot! Fixed